sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/arithmetic.scala

```diff
@@ -298,7 +298,7 @@ abstract class BinaryArithmetic extends BinaryOperator
           case "+" => "try_add"
           case "-" => "try_subtract"
           case "*" => "try_multiply"
-          case _ => ""
+          case _ => "unknown_function"
```

Comment:
In which cases can't we suggest anything? For which functions are there no `try_*`?

Reply:
For now there is no case, but it seemed weird to throw another error within error parsing of the initial error and hide the actual cause of the error.

Reply:
I see, lgtm.